drivers/virtualbox/virtualbox.go

```diff
@@ -452,19 +452,26 @@ func (d *Driver) GetIP() (string, error) {
 	if s != state.Running {
 		return "", drivers.ErrHostIsNotRunning
 	}
-	cmd, err := drivers.GetSSHCommandFromDriver(d, "ip addr show dev eth1")
+	output, err := drivers.RunSSHCommandFromDriver(d, "ip addr show dev eth1")
 	if err != nil {
 		return "", err
 	}
 
-	// reset to nil as if using from Host Stdout is already set when using DEBUG
-	cmd.Stdout = nil
+	var buf bytes.Buffer
+	if _, err := buf.ReadFrom(output.Stdout); err != nil {
+		return "", err
+	}
 
-	b, err := cmd.Output()
+	n, err := buf.Read([]byte{1})
```

Comment:
this is not how to initialize a slice of 1 element :) You're creating a slice here with a SOH byte initialized with it.  Please use:

    make([]byte, 1)